spacy_llm/tasks/util/examples.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, List
+
+from pydantic import BaseModel
+
+
+class SpanExample(BaseModel):
+    text: str
+    entities: Dict[str, List[str]]
+
+
+class LemmaExample(BaseModel):
+    text: str
+    lemmas: List[Dict[str, str]]
+
+
+class SpanReason(BaseModel):
+    text: str
+    is_entity: bool
+    label: str
+    reason: str
+
+    @classmethod
+    def from_str(cls, s: str, sep: str = "|"):
```

Comment:
This would benefit from a docstring and some illustrative examples demonstrating the expected input.